logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -43,16 +42,16 @@ sealed class Event(open val id: String) {
             override val id: String,
             override val conversationId: ConversationId,
             val addedBy: UserId,
-            val members: ConversationMembers,
-            val from: String
+            val members: List<Member>,
+            val time: String
```

Comment:
would this be a long timestamp or a `formatedTime` String? In case of the latter, I think it would be useful to be slightly more descriptive on the naming.